app/routes/view-application.js

```diff
@@ -36,6 +36,12 @@ module.exports = {
       }
       const applicationHistory = await getApplicationHistory(request.params.reference)
 
+      const claimDataStatus = ['IN CHECK', 'REJECTED', 'READY TO PAY']
+      let applicationEvents
+      if ((application?.claimed || claimDataStatus.includes(application?.status?.status)) && !application?.data?.dateOfClaim) {
+        applicationEvents = await getApplicationEvents(application?.data?.organisation.sbi)
```

Comment:
wondering if we need to check on the reference number to future proof this to ensure we dont check for a different agreement for the same SBI?

Reply:
This call retrieves all events for am individual SBI. The model code the  filters the events down to a single claim-claimed event. I'll add the application reference in as qn additional filter.

Reply:
@liamwilson-kainos I've taken a look at this this morning and the claim-claimed event doesn't include the application reference, any thoughts on how this can be filtered at this stage?

Reply:
Resolving this comment as it is not possible to have multiple applications for an SBI right now. By the time we introduce this capability we will be using the data in the application table.